src/___tests___/taskFunctions.test.js

```diff
@@ -0,0 +1,28 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Please kindly fix the eslint error you are getting rather than disabling it 👍 
- [ ] I want to also suggest you add more test cases to the ones you have, each for the add and remove functions, also you can add test cases that covers edge cases like when a user tries to add a task without inputing any text in the provided fields, which can be represented by using an empty string `" "`